Formula/balena-cli.rb

```diff
@@ -19,11 +19,12 @@ class BalenaCli < Formula
     sha256 high_sierra: "0e670f3cc482565ddc1a166316407fab2fd008f12128a06f7f73c3bcf07a641d"
   end
 
-  depends_on "node"
+  depends_on "node@14"
```

Comment:
Will note that previous attempts and upstream docs say to use `node@12`.

I was able to build with `node@14` (failed with `node@16`). 

Older PR attempts probably failed due to either:
1. Parallelizing build, which fails on `ronomon`
2. Possibly not updating PATH in test, which will use default shim `/usr/bin/env node`.